### PR TITLE
#1 - Setup Laravel project

### PR DESCRIPTION
**Pull request by AmonDeShir**
I set up a new project.

### CODE REVIEW

**Comment by kamilpiech97** [comment]
Title checker failed.

**Comment by kamilpiech97** [comment]
@Blusia lets add renovate at start

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+@blumilksoftware/blumilk
```
Comment (EwelinaSkrzypacz) [inline .github/CODEOWNERS:1]:
```suggestion
@blumilksoftware/interns2024b
```

Comment (EwelinaSkrzypacz) [reply .github/CODEOWNERS:1]: Or maybe better - instead of this, please add me and @kamilpiech97 as code owners

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. Please check GitHub annotations and fix this

image

**Review by kamilpiech97 — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**